### DISCUSSION TITLE
Starting root_mariadb_1 ... error

### DISCUSSION THREAD

**Discussion by KatzeMau** [photoprism/photoprism#2711] [Q&A]
When i do docker-compose up it downloads the images but cant start them.
I tried googling the answer but sadly no success. 

Digest: sha256:c8f817ab07d329fc9ff888fd19b0c5718d177411132c4d7b3e9072a02725f98f
Status: Downloaded newer image for mariadb:10.8
Starting root_mariadb_1 ... error

ERROR: for root_mariadb_1  Cannot start service mariadb: OCI runtime create failed: container_linux.go:367: starting container process caused: process_linux.go:495: container init caused: process_linux.go:458: setting cgroup config for procHooks process caused: can't load program: operation not permitted: unknown

ERROR: for mariadb  Cannot start service mariadb: OCI runtime create failed: container_linux.go:367: starting container process caused: process_linux.go:495: container init caused: process_linux.go:458: setting cgroup config for procHooks process caused: can't load program: operation not permitted: unknown
ERROR: Encountered errors while bringing up the project.
root@PhotoPrism2:~#

**Comment by lastzero** [comment]
Try to update Docker and make sure you have all Docker (kernel) dependencies installed, like for "cgroups". From what I know, these are commonly used to limit resources (CPU, RAM,...). You should therefore also check if you have added custom settings that are now causing problems.
